Step-by-step-Tutorials/A-Service-is-a-Collection-of-Communicating-Actors.md

```diff
@@ -92,7 +92,7 @@ public class Manager : Orleans.Grain, IManager
 {
     public override Task ActivateAsync()
     {
-        _me = GrainFactory.GetGrain<IEmployee>(this.GetPrimaryKey());
```

Comment:
From within grain code it should be `this.GrainFactory.GetGrain<>` or `GrainFactory.GetGrain<>`, not `this.GetGrain<>` which won't compile.